examples/platformer/platformer.cpp

```diff
@@ -125,40 +144,121 @@ struct Player {
   }
 
   void update() {
-    static float max_speed_x = 2.0f;
     static float ground_acceleration_x = 0.5f;
     static float air_acceleration_x = 0.2f;
+    static float ground_drag_x = 0.70f;
+    static float air_drag_x = 0.8f;
     static float duration = 0.01f;
-    static Vec2 gravity(0, 9.8f * 5.0f);   // normal gravity is boring!
+    static float jump_velocity = 4.0f;
+    static Vec2 gravity(0, 0.98f / 10.0f);
 
-    if (pressed(Button::DPAD_LEFT)) {
-      vel.x = vel.x - (on_ground() ? ground_acceleration_x : air_acceleration_x);
-    }
+    bool is_on_ground = on_ground();
+    state == STILL;
```

Comment:
Bit late, but I think _every_ build threw a warning on that. Also a couple of double -> float conversions below.

Reply:
D'oh! I'm pretty sure I saw this warning and forgot to fix it, too. It's... been hectic!